apps/cowswap-frontend/src/modules/tradeFormValidation/hooks/useTradeFormValidationContext.ts

```diff
@@ -30,7 +30,7 @@ export function useTradeFormValidationContext(): TradeFormValidationCommonContex
   const { isSupportedWallet } = useWalletDetails()
   const gnosisSafeInfo = useGnosisSafeInfo()
 
-  const isSafeReadonlyUser = gnosisSafeInfo?.isReadOnly || false
+  const isSafeReadonlyUser = !!gnosisSafeInfo?.isReadOnly
```

Comment:
Convert falsy values to false directly

Reply:
fine, but i have a light preference for  `gnosisSafeInfo?.isReadOnly || false` as I find it easier to understand (as this is returning a boolean, you have to think not not is the same as the bookean, then for the undefined will bee, false

Alternatively, you could do `gnosisSafeInfo?.isReadOnly === true` which is probably better than my suggestion above